portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,6 +37,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");//going to priint directly on page
     response.getWriter().println(json);
     }
+  
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+    comments.add("I like your pictures!");
+    String name = request.getParameter("name-input");
+    String text = request.getParameter("text-input");
+    response.setContentType("text/html;");
+    comments.add(text);
+    response.getWriter().println(comments);
+  }
 
   private String convertToJson(ArrayList<String> list){
     String json = "['";
```

Comment:
You could use Gson library here instead of hand-creating this json string.

see: https://stackoverflow.com/questions/27893342/how-to-convert-list-to-a-json-object-using-gson

Reply:
Changed to Gson but kept the manual conversion just in case. Everything should be using gson now.